Makefile

```diff
@@ -17,4 +17,5 @@ lint-fix:
 	@docker-compose --file=$(lint_docker_compose_file) run gin-golinter golangci-lint run --fix \
 		&& echo "✔️  fixed without errors" \
 		|| (echo "⚠️️  you need to fix above issues manually" && exit 1)
-	@echo "⚠️️ run \"make lint-check\" again to check what did not fix yet"
\ No newline at end of file
+	@echo "⚠️️ run \"make lint-check\" again to check what did not fix yet"
```

Comment:
What is the change here?

Reply:
nothing sir I think it picked up empty space.